docker/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '2'
+services:
```

Comment:
Nitpicking... might be easier to read if you split each service by an empty line.